pages/application.js

```diff
@@ -0,0 +1,82 @@
+import styled from 'styled-components';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+    faFacebook,
+    faInstagram,
+    faLinkedinIn,
+    faMediumM,
+    faTwitter,
+    faYoutube,
+  } from '@fortawesome/free-brands-svg-icons';
+import Head from 'next/head';
+
+import JoinUs from '../components/JoinUs';
+
+const ApplicationPageContainer = styled.div`
+  background: ${p => p.theme.colors.background};
+  height: fit-content;
+  position: relative;
+`;
+
+const BackgroundImage = styled.img`
+  position: absolute;
+  width: 510px;
+  right: 0;
+`;
+
+const FooterContainer = styled.div`
+    text-align: center;
+    padding: 64px 0;
+    width: 100%;
+    color: ${p => p.theme.colors.secondary};
+`;
+
+const SocialMediaIcons = styled.div`
+  font-size: 32px;
+  margin-bottom: 16px;
+
+  a {
+    margin: 0 32px;
+  }
+`;
+
+
+export default function Charcuterie() {
+  return (
+    <>
+      <Head>
+        <title>Application Page | nwPlus</title>
+      </Head>
+
+      <main>
+        <ApplicationPageContainer>
+            <BackgroundImage src="/assets/logos/trying.svg" alt="an astronaut reaching inside a window"/>
+            <JoinUs hiring={true} deadlineDay={30} deadlineMonth="September"/>
+            <FooterContainer>
+                <SocialMediaIcons>
+                    <a href="https://www.facebook.com/nwplusubc" target="_blank" rel="noreferrer">
+                    <FontAwesomeIcon icon={faFacebook} />
+                    </a>
+                    <a href="https://www.instagram.com/nwplusubc" target="_blank" rel="noreferrer">
+                    <FontAwesomeIcon icon={faInstagram} />
+                    </a>
+                    <a href="https://twitter.com/nwplusubc" target="_blank" rel="noreferrer">
+                    <FontAwesomeIcon icon={faTwitter} />
+                    </a>
+                    <a href="https://www.linkedin.com/company/nwplus" target="_blank" rel="noreferrer">
+                    <FontAwesomeIcon icon={faLinkedinIn} />
+                    </a>
+                    <a href="https://www.youtube.com/c/nwPlusUBC" target="_blank" rel="noreferrer">
+                    <FontAwesomeIcon icon={faYoutube} />
+                    </a>
+                    <a href="https://medium.com/nwplusubc" target="_blank" rel="noreferrer">
+                    <FontAwesomeIcon icon={faMediumM} />
+                    </a>
+                </SocialMediaIcons>
+                <div>Copyright &copy; 2021 nwPlus</div>
+            </FooterContainer>
+        </ApplicationPageContainer>
```

Comment:
I took a look at the designs and I think the background should also have some noise (grainy texture). I had something similar in Background.js implemented, but the lines were coupled with the noise. I think you can easily decouple them by copying this previous commit: https://github.com/nwplus/www/pull/49/commits/275ff09924a8e99c7d8d121d7e804871209c5b91#diff-647d05634d5e0a2a5a11a5dfe3b670215ea842eb25995dbeca4114d402123611 
Lmk if you need any help with this!